yetanotherco/aligned_layer
# feat(operator): add basic event susbscription and listening flow

This PR adds basic operator flow:
* receive config from cli
* create operator instance from config
* events listening loop

To test it:
* start anvil
* register the operator with `make operator-full-registration`
* start the operator with `make operator-start`
* In another terminal, send a proof with `make send-plonk-proof`

You should see the task's index logged by the operator, which received the new task event from the contract
